types/express-brute/index.d.ts

```diff
@@ -25,7 +25,7 @@ declare class ExpressBrute {
      * @param {Function}    next        The next middleware.
      * @return {RequestHandler} The Request handler.
      */
-    prevent(request: express.Request, response: express.Response, next: express.NextFunction): express.RequestHandler;
```

Comment:
wrong type — it's actually RequestHandler
1. see https://github.com/AdamPflug/express-brute/blob/ab4b4e1ef1277c702f5f02efc8fbe39ba6b12b68/index.js#L32 - call
2. see https://github.com/AdamPflug/express-brute/blob/ab4b4e1ef1277c702f5f02efc8fbe39ba6b12b68/index.js#L46 - returns function
therefore, `prevent` is assigned with RequestHandler

Reply:
fixing types here allows to minimize changes to several other packages that depend on it having failing tests